### PR TITLE
stim fix

### PR DESCRIPTION
**Pull request by AdamElTablawy**
fixes stim

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

:cl:
tweak: fixes stims
/:cl:

### CODE REVIEW

**Comment by buffyuwu** [comment]
oh you- made a whole new PR close the other lol

**Comment by AdamElTablawy** [comment]
its testmergable again @buffyuwu

File: code/modules/fallout/reagents/medicines.dm [hunk 1]
```diff
@@ -7,6 +7,7 @@
 	metabolization_rate = 0.5 * REAGENTS_METABOLISM
```
Comment (buffyuwu) [inline code/modules/fallout/reagents/medicines.dm:7]: this should be higher

**Review by buffyuwu — CHANGES_REQUESTED** [review]

File: code/modules/fallout/reagents/medicines.dm (hunk 1 above)
Comment (buffyuwu) [reply code/modules/fallout/reagents/medicines.dm:7]:
```suggestion
	metabolization_rate = 5 * REAGENTS_METABOLISM
```

**Pull request closed without merging**